server/sonar-web/src/main/js/apps/projects/filters/Filter.tsx

```diff
@@ -143,10 +137,9 @@ export default class Filter extends React.PureComponent<Props> {
     const { options, highlightUnder } = this.props;
     if (options && options.length > 0) {
       if (highlightUnder != null) {
-        const max = this.props.highlightUnderMax || options.length;
         const beforeHighlight = options.slice(0, highlightUnder);
-        const insideHighlight = options.slice(highlightUnder, max);
-        const afterHighlight = options.slice(max);
+        const insideHighlight = options.slice(highlightUnder, highlightUnder + 1);
+        const afterHighlight = options.slice(highlightUnder + 1);
```

Comment:
If my understanding is correct, both `highlightUnder` and `highlightUnderMax` can be dropped from props, because each facet is now using the same default selection behavior.

Reply:
Indeed, all the `...highlight...` stuff should be dropped or renamed because it's there to handle cumulative facets highlighting. Same for the related CSS that should be cleaned too.